nilearn/plotting/html_stat_map.py

```diff
@@ -49,11 +50,12 @@ def _data_to_sprite(data):
 
 def _threshold_data(data, threshold=None):
     """ Threshold a data array.
-        Returns: data (masked array), threshold (updated)
+        Returns: data (array), mask (boolean array) threshold (updated)
```

Comment:
It's True where the original values are kept. This is identical behavior, see:

```python
import numpy as np
data = np.arange(7)
threshold = 2

new = [data > threshold]
old = np.ma.masked_where(data > threshold, data)
assert np.all(new == old.mask)
```

Reply:
> ```python
> old = np.ma.masked_where(data > threshold, data)
> ```

this is masked where the condition is True:
```
>>> np.ma.masked_where(np.arange(5) > 2, np.arange(5))                                                                                            
masked_array(data=[0, 1, 2, --, --],
             mask=[False, False, False,  True,  True],
       fill_value=999999)
```

Reply:
Yes, sorry, I tried to edit my comment to clarify -- I realized "masked" might not be the best term since there's the masked data type (`--`) vs where the mask is applied ! See the updated comment :)